api/lib/OCR.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+
+const gs = require('ghostscript4js');
+const tesseract = require('node-tesseract');
+
+const errorMessages = require('./error-messages');
+
+function OCR(options) {
+    if (typeof options === "object") {
+        const acceptedKeys = Object.keys(this.options);
+        Object.keys(options).forEach((val) => {
+            if (!acceptedKeys.includes(val)) return delete options[val];
+            if (typeof this.options[val] !== typeof options[val]) {
+                throw Error(errorMessages.TYPE_MISMATCH_ERROR(val, typeof this.options[val], typeof options[val]));
+            }
+        });
+        this.options = Object.assign(this.__proto__.options, options);
+    }
+}
+
+OCR.prototype.options = {
+    l: 'por',
+    psm: 6,
+    binary: '/usr/bin/tesseract'
+};
+
+OCR.prototype.getText = (file) => {
+    file = path.dirname(require.main.filename) + '/' + file;
```

Comment:
This assumes the file is in the same folder as the OCR lib right?
Maybe it's better to receive a file_path instead and not assume a location...